ui/single/task-priority-too-high.stderr

```diff
@@ -1,9 +1,13 @@
+warning: unused import: `rtfm::app`
+ --> $DIR/task-priority-too-high.rs:3:5
+  |
+3 | use rtfm::app;
```

Comment:
Uh, spurious warning? Is this a known rustc bug?

Reply:
The warning it's also present on 1.36.0. Perhaps this has been there for a while but compiletest ignores warnings by default?

Reply:
Yeah, that's probably what's happening